src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -287,6 +288,48 @@ Adapter                  @astrojs/vercel/serverless
 Integrations             none
 ```
 
+## `astro preferences`
+
+Manage user preferences with the `astro preferences` command. User preferences are specific to individual Astro users, unlike the `astro.config.mjs` file which changes behavior for everyone working on a project.
+
+User preferences are scoped to the current project by default, stored in a local `.astro/settings.json` file. Using the `--global` flag, user preferences can also be applied to every Astro project on the current machine. Global user preferences are stored in an operating system-specific location.
+
+For example, to disable the devToolbar in a specific Astro project:
+
+```shell
+astro preferences disable devToolbar
+```
+
+To disable the devToolbar in all Astro projects on the current machine:
+
+```shell
+astro preferences disable --global devToolbar
+```
+
+The devToolbar can later be enabled with:
+
+```shell
+astro preferences enable devToolbar
+```
+
+The `reset` command resets a preference to its default value:
+
+```shell
+astro preferences reset devOverlay
+```
+
+The `list` command prints all current user preferences. It also supports a machine-readable `--json` output.
+
+```shell
+astro preferences list
+```
+
+### Available preferences
+
+Currently, `devToolbar` is the only available preference. The full list of [default preferences](https://github.com/withastro/astro/blob/main/packages/astro/src/preferences/defaults.ts) can be found in the Astro repository.
+
+In the future, we plan to add more preferences to personalize your Astro experience.
```

Comment:
Am I remembering correctly that we try _not_ to document anything that we plan to do in the future?